sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -273,7 +274,8 @@ def __init__(self,
     ]  # type: List[Tuple[TransformExecutor, Timestamp]]
     self._counter_factory = counters.CounterFactory()
     self._metrics = DirectMetrics()
-
+    self._test_stream_event_stub = test_stream_event_stub
+    self._test_stream_event_channel = None
```

Comment:
Similarly, the test_stream_event_channel doesn't seem a property of the evaluation context, rather a property of the test stream (evaluator) itself.

Reply:
Ack, I moved the event generation into the test_stream_impl module.